packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -114,7 +115,13 @@ export const waitForCallCompositeToLoad = async (page: Page): Promise<void> => {
   await page.bringToFront();
   await page.waitForLoadState('load');
   await waitForPageFontsLoaded(page);
-  const startCallButton = await waitForSelector(page, dataUiId('call-composite-start-call-button'));
+
+  // Tests often timeout at this point because device enumeration can take > 5 seconds.
```

Comment:
I'm OK bumping the total overall timeout we wait for all selectors to 10 seconds, the assumption being that most are immediate so only failed tests will trigger the 10s timeout

Reply:
We should start with targeted bumps.

Besides being a more pointed hammer, it gives us obvious targets to bring the timeouts back down. A blanket bump means we have no idea where the timeouts are adding more latency.

In my experience, timeouts and retry config should be as specific as possible.